acm/src/main/java/org/literacybridge/acm/config/DBConfiguration.java

```diff
@@ -165,22 +153,33 @@ public void init() {
             controlAccess = new ControlAccess(this);
             controlAccess.init();
 
+            final Taxonomy taxonomy = Taxonomy.createTaxonomy(sharedACMDirectory);
+            if (!AudioItemIndex.indexExists(getLuceneIndexDirectory())) {
```

Comment:
What about the case of a brand new ACM? Wouldn't that case not have a Lucene Index? In which case there would be nothing to migrate.